### DISCUSSION TITLE
DITA-OT 4.2.1 Apache FOP Version SVN and PDF document size is greater

### DISCUSSION THREAD

**Discussion by FlinkRonny** [dita-ot/dita-ot#4423] [Q&A] [answered]
Hi!
I just migrated a plugin from DITA-OT 3.7.3 to DITA-OT 4.2.1.
The generated PDF is identical as before with 3.7.3, same content and the same number of pages.
Except one difference is that the file size is 2MB more for the PDF generated with DITA-OT 4.2.1.
When right-clicking the in the PDF and select document properties, it shows Apache FOP Version SVN.
Shouldn't it be Apache FOP Version 2.9?

**Comment by raducoravu** [comment, marked as answer]
The DITA OT does not have much control over the internal changes the third party Apache FOP goes through.
About the document properties, I added an issue on the Apache FOP list: https://issues.apache.org/jira/browse/FOP-3169
About the increase in PDF size, maybe you can ask around on the FOP mailing list: https://xmlgraphics.apache.org/fop/maillist.html
Maybe there is some sort of setting which can be set in the "DITA-OT/plugins/org.dita.pdf2.fop/cfg/fop.xconf" FOP config file to reduce the PDF.
The default fop xconf does not seem to enable accessibility (which usually increases the PDF size).

**Comment by infotexture** [comment]
As @raducoravu said, the `SVN` string in the document properties is apparently a known issue with FOP 2.9.

It was reported in [FOP-3146](https://issues.apache.org/jira/browse/FOP-3146) and [FOP-3169](https://issues.apache.org/jira/browse/FOP-3169) and resolved in https://github.com/apache/xmlgraphics-fop/commit/fa89ae7b7a349d7f015b6eeb650a839979a8eed0, so it should be fixed in the next version.

**Comment by FlinkRonny** [comment]
Thanks Radu and Roger for your replies.
I appreciate your insights.
The document properties  issue and file size issue won’t create any problems for us, but good to know, that I didn’t miss anything or do something wrong when migrating the plugin.
